Dnn.CommunityForums/CustomControls/UserControls/TopicView.cs

```diff
@@ -1370,6 +1371,22 @@ private string ParseContent(DataRow dr, string tempate, int rowcount)
                 sbOutput.Replace("[ACTIONS:DELETE]", string.Empty);
             }
 
+            if (ForumUser.IsAdmin || ForumUser.IsSuperUser)
```

Comment:
@johnhenley @Timo-Breumelhof Should this also include Moderators? 🤔 RIght now, it's only superuser and Admins.

Reply:
By the way, maybe it's only SUs and Admins now, but a setting can be added later to allow Moderators too.  🤷🏽‍♂️  I'm open to either idea.

Reply:
> By the way, maybe it's only SUs and Admins now, but a setting can be added later to allow Moderators too. 🤷🏽‍♂️ I'm open to either idea.

Since this functionality disables a user, I assumed it needed to be an administrator or super-user. Do moderators have this ability?
![image](https://github.com/DNNCommunity/Dnn.CommunityForums/assets/9553126/7f4f6880-0bfa-48a7-874d-eb0ff31e6d01)

Reply:
> > By the way, maybe it's only SUs and Admins now, but a setting can be added later to allow Moderators too. 🤷🏽‍♂️ I'm open to either idea.
> 
> Since this functionality disables a user, I assumed it needed to be an administrator or super-user. Do moderators have this ability? ![image](https://user-images.githubusercontent.com/9553126/267798433-7f4f6880-0bfa-48a7-874d-eb0ff31e6d01.png)
We can adapt the code so moderators can remove the posts and remove them from functioning in the forums but only if they are an admin or superuser do the additional step of disabling the user. I don't think we can assume that a moderator has the admin rights to disable the user outright.

Reply:
I think it could be nice to have that as an option for Moderators?
It does seem to save a lot of work for an admin) when you have a group of trusted moderators?

Reply:
Personally I'm fine with it being available to moderators. What if we use this modUser permission? ![image](https://github.com/DNNCommunity/Dnn.CommunityForums/assets/9553126/111439a3-7c35-44aa-9abf-8ecd04d468fd)